src/core/lib/event_engine/event_engine_common.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains definitions that are common to all event engine
+// implementations.
+
+#include <grpc/support/port_platform.h>
+
+#include <grpc/event_engine/event_engine.h>
+#include <grpc/event_engine/port.h>
+#include <grpc/support/log.h>
+
+#include "src/core/lib/gprpp/sync.h"
+
+namespace grpc_event_engine {
+namespace experimental {
+
+EventEngine::ResolvedAddress::ResolvedAddress(const sockaddr* address,
```

Comment:
What do you think about calling this `src/core/lib/event_engine/resolved_address.cc`? The original `event_engine.cc` was a small catch-all file (2 concepts within), and the intent was to eventually reorganize code in there once the EventEngineFactory story was outlined better. Now that it's just ResolvedAddress here, I think it makes sense to organize it as such. Similar to `channel_args_endpoint_config`, maybe.

Reply:
Thanks for taking a look. Fixed in latest commit